File: frontend/upwork-client/src/app/models/user-absence.model.ts
import { AbsenceType } from './enums/absence-type.enum';

export interface UserAbsence {
  id: string;
  organizationId: string;
  firstName: string;
  lastName: string;
  fromDate: Date;
  toDate: Date;
  absenceType: AbsenceType;
}

File: backend/UpWork/UpWork.Common/Models/UserAbsenceModel.cs
using System;
using UpWork.Common.Enums;

namespace UpWork.Common.Models
{
	public class UserAbsenceModel
	{
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public Guid OrganizationId { get; set; }
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
        public AbsenceTypes AbsenceType { get; set; }
    }
}

